### PR TITLE
Add Internationalization for Português (Brasil)

### PR DESCRIPTION
**Pull request by sangyuxiaowu**

### CODE REVIEW

File: src/main/services/i18n/language.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export const language = {
   en: 'English',
   es_ES: 'Español',
+  pt_BR: 'Português (Brasil)',
```
Comment (antonreshetov) [inline src/main/services/i18n/language.ts:4]: I think "Português (do Brasil)" is more correct. For example, that's what appears when you select a language at https://developer.mozilla.org/.

Comment (sangyuxiaowu) [reply src/main/services/i18n/language.ts:4]: > I think "Português (do Brasil)" is more correct. For example, that's what appears when you select a language at https://developer.mozilla.org/.

It should be almost the same, I refer to Microsoft's.
https://docs.microsoft.com/zh-cn/locale/?target=https%3A%2F%2Fdocs.microsoft.com%2Fzh-cn%2Fdotnet%2F

Comment (antonreshetov) [reply src/main/services/i18n/language.ts:4]: Then it doesn't matter, it's probably a matter of preference )

Comment (sangyuxiaowu) [reply src/main/services/i18n/language.ts:4]: This is up to you to decide. I have no opinion.

**Pull request merged**